packages/bot/src/scratch/hooks/flyout_base.js

```diff
@@ -80,19 +80,23 @@ Blockly.Flyout.prototype.placeNewBlock_ = function(event, oldBlock) {
   
     // The offset in pixels between the flyout workspace's origin and the upper
     // left corner of the injection div.
-    const flyoutOffsetPixels = this.workspace_.getOriginOffsetInPixels();
+    // const flyoutOffsetPixels = new goog.math.Coordinate(250, 0);
```

Comment:
@sam-binary do we need to comment these, can we remove it if we wont need it anymore